### ISSUE TITLE
Rename `Camera` objects

### ISSUE THREAD

**Issue by cabanier** [immersive-web/model-element#11]
`camera` implies that this feature controls the projection matrices of the model.
If <model> is implemented in a VR device, `HTMLModelElementCamera` wouldn't control the camera but the orientation.

**Comment by bialpio** [comment]
Related question: in a VR / AR device, would this be something that the page will be able to control or even access? I imagine that in non-seated VR / AR, the UA would "pop out" the model into the user's virtual / real world and the user would be able to walk around the model. If this is something that the page can inspect, it'd be a way for it to learn about the user's pose, or am I missing something here?

**Comment by cabanier** [comment]
Yes, it would pop out but it seems that the author would still want the ability to modify the orientation.

**Comment by zachernuk** [comment]
@cabanier I feel like the `entityTransform` we've discussed probably closes this - is it good to close?
